src/components/Filter/TodoFilter.tsx

```diff
@@ -0,0 +1,71 @@
+import classNames from 'classnames';
+import { useState } from 'react';
+import { FilterBy } from '../../types/FilterType';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  setSortBy: (value: FilterBy) => void;
+  clearAllCompleted: () => void;
+  completedTodo: Todo[];
+};
+
+export const TodoFilter: React.FC<Props> = ({
+  setSortBy,
+  clearAllCompleted,
+  completedTodo,
+}) => {
+  const [selectedTab, setSelectedTab] = useState('all');
+
+  return (
+    <>
+      <nav className="filter" data-cy="Filter">
+        <a
+          data-cy="FilterLinkAll"
+          href="#/"
+          className={classNames('filter__link',
+            { 'filter__link selected': selectedTab === 'all' })}
```

Comment:
create enum for filters 
```suggestion
            { 'filter__link selected': selectedTab === Frilters.All })}
```